### PR TITLE
Issue #1204 Document alarms API for child devices

### PR DESCRIPTION
**Pull request by albinsuresh**
## Proposed changes

Document alarms API for child devices

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Improvement (general improvements like code refactoring that doesn't explicitly fix a bug or add any new functionality)
- [x] Documentation Update (if none of the other choices apply)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Paste Link to the issue
<br/>

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/thin-edge/thin-edge.io/blob/main/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://github.com/thin-edge/thin-edge.io/blob/main/CONTRIBUTOR-LICENSE-AGREEMENT.md) (in all commits with git commit -s)
- [ ] I ran `cargo fmt` as mentioned in [CODING_GUIDELINES](https://github.com/thin-edge/thin-edge.io/blob/main/CODING_GUIDELINES.md)
- [ ] I used `cargo clippy` as mentioned in [CODING_GUIDELINES](https://github.com/thin-edge/thin-edge.io/blob/main/CODING_GUIDELINES.md)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: docs/src/tutorials/raise-alarm.md [hunk 1]
```diff
@@ -72,6 +72,12 @@ An already raised alarm can be cleared by sending an empty message with retained
 
 If alarms of different severities exist for a given alarm type, they must all be cleared separately as they're all treated as independent alarms.
 
+### Raising alarms from child devices
+
+Events for child devices can be sent by publishing the event payload to `tedge/alarms/<severity>/<alarm-type>/<child-device-id>` topic,
```
Comment (didier-wenzek) [inline docs/src/tutorials/raise-alarm.md:77]:
```suggestion
Alarms for child devices can be sent by publishing the alarm payload to `tedge/alarms/<severity>/<alarm-type>/<child-device-id>` topic,
```

**Review by PradeepKiruvale — APPROVED** [review]
LGTM

File: docs/src/tutorials/raise-alarm.md [hunk 2]
```diff
@@ -72,6 +72,12 @@ An already raised alarm can be cleared by sending an empty message with retained
 
 If alarms of different severities exist for a given alarm type, they must all be cleared separately as they're all treated as independent alarms.
 
+### Raising alarms from child devices
```
[the remark this answers is not in the document]
Comment (albinsuresh) [reply docs/src/tutorials/raise-alarm.md:75]: The term "child device" doesn't imply external as it covers both external as well as logical child devices as well. But the alarms contract is the same, irrespective of the type of child device. So, I hope the generic term "child device" is sufficient here.

**Comment by matthiasbeyer** [comment]
`Signed-off-by` line is missing.

**Pull request merged**